internal/services/configstore/api/org.go

```diff
@@ -320,3 +320,161 @@ func (h *OrgMembersHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.log.Err(err).Send()
 	}
 }
+
+type OrgInvitationsHandler struct {
```

Comment:
Since you have create an invitation.go action (both in configstore and gateway) you should do the same for api package or just move everything under user.go and org.go like in api package.

Reply:
ok I move all under user.go and org.go